### PR TITLE
generating docker image

### PR DESCRIPTION
**Pull request by nilminiwso2**
- replace image as it was outdated
- formatter other changes

### CODE REVIEW

File: en/micro-integrator/docs/develop/generate-docker-image.md [hunk 1]
```diff
@@ -4,52 +4,102 @@ See the topics given below.
...
+
+        3.  Click **Next**. Select the **Config** projects that you want to include in the Docker image and click **Finish**.  
+
+            ![Create docker image](../assets/img/create_project/select_artifact_docker.png)  
+        
+            Once the Docker image is successfully created, a message similar to the following appears in your screen.  
+
+            ![Create docker image](../assets/img/create_project/docker_image_successful.png)
+
+    -   **Generate Docker Image with an Existing Project**
```
Comment (nilminiwso2) [inline en/micro-integrator/docs/develop/generate-docker-image.md:102]: @rosensilva - When I tried this, nothing happened. Is there suppose to be some notification? Is this similar to the 'Build' option for the 'Docker Exporter Project'?

File: en/micro-integrator/docs/develop/generate-docker-image.md [hunk 2]
```diff
@@ -4,52 +4,102 @@ See the topics given below.
...
+                        The version of the composite
+                        application.
+                    </td>
+                </tr>
+                <tr>
+                    <td>
+                        Name of the Docker Image
+                    </td>
+                    <td>
+                        Give a name for the Docker image.
```
Comment (nilminiwso2) [inline en/micro-integrator/docs/develop/generate-docker-image.md:71]: @rosensilva - could you provide the format for this? Shouldnt it be 'docker-name/repository-name'? Tried but didnt work.

File: en/micro-integrator/docs/develop/generate-docker-image.md (hunk 1 above)
Comment (rosensilva) [reply en/micro-integrator/docs/develop/generate-docker-image.md:102]: I might take some time in the first time, did you wait about 10 mins?

File: en/micro-integrator/docs/develop/generate-docker-image.md (hunk 2 above)
Comment (rosensilva) [reply en/micro-integrator/docs/develop/generate-docker-image.md:71]: you can give a name you prefer for the docker image. @sajithaliyanage please confirm

File: en/micro-integrator/docs/develop/generate-docker-image.md (hunk 1 above)
Comment (nilminiwso2) [reply en/micro-integrator/docs/develop/generate-docker-image.md:102]: Didn't wait for 10 minutes. :) But now I added an image with the expected notification. I am guessing the docker image will get created and we will receive the 'Docker image successfully created' message?

**Pull request merged**